dojo/unittests/tools/test_nuclei_parser.py

```diff
@@ -0,0 +1,130 @@
+from django.test import TestCase
+from dojo.tools.nuclei.parser import NucleiParser
+from dojo.models import Test
+
+
+class TestNucleiParser(TestCase):
+
+    def test_parse_no_findings(self):
+        testfile = open("dojo/unittests/scans/nuclei/no_findings.json")
+        parser = NucleiParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(0, len(findings))
+
+    def test_parse_many_findings(self):
+        testfile = open("dojo/unittests/scans/nuclei/many_findings.json")
+        parser = NucleiParser()
+        findings = parser.get_findings(testfile, Test())
+        testfile.close()
+        for finding in findings:
+            for endpoint in finding.unsaved_endpoints:
+                endpoint.clean()
+
+        self.assertEqual(9, len(findings))
+
+        with self.subTest(i=0):
+            finding = findings[0]
+            self.assertEqual("OpenSSH 5.3 Detection", finding.title)
+            self.assertEqual("Low", finding.severity)
+            self.assertEqual(1, finding.nb_occurences)
+            self.assertIsNotNone(finding.description)
+            self.assertEqual("network,openssh", finding.unsaved_tags)
+            self.assertIsNotNone(finding.references)
+            self.assertEqual("nuclei-example.com", finding.unsaved_endpoints[0].host)
```

Comment:
Add checks on the number of endpoints please.

Reply:
Added